checks/yum_installed_linux.go

```diff
@@ -25,18 +23,24 @@ func init() {
 // Arguments:
 //   package (required): A string value that represents the rpm package
```

Comment:
nit: doc should also be updated.

Reply:
These docs seem like they were ported from Python because they aren't necessary in a language like Go with its own docs generation and static typing. A comment on what the struct is used for, sure, but the rest of them should probably be refactored away.